docs/event-bus/docs/02-01-event-bus.md

```diff
@@ -2,48 +2,53 @@
 title: Architecture
 ---
 
-See the diagram and steps for an overview of the basic Event Bus flow:
+When you create a lambda or a service to perform a given business functionality, you must define which Events trigger it. Define triggers by creating the Subscription CR in which you instruct the Event Bus to forward the Events of a particular type to your lambda. 
```

Comment:
We could possibly move this inside Event consumption as the context is closer to that.

Reply:
Moved:)